src/app/LoginController.java

```diff
@@ -75,10 +84,11 @@ public void run() {
     * @param ActionEvent when button clicked
     */
    public void login(ActionEvent event) throws IOException {
-      boolean isLoggedIn = true;
-
+      Account account = loginDao.getAccountByLoginName(username.getText());
+      boolean isLoggedIn = account!=null && account.validatePassword(password.getText());
       if (isLoggedIn) {
```

Comment:
This could just be `account!=null && account.validatePassword(password.getText())` for simplicity, but no big deal.